test/onnx/test_fx_op_consistency.py

```diff
@@ -787,6 +787,7 @@ class TestOnnxModelOutputConsistency(onnx_test_common._TestONNXRuntime):
         "native_batch_norm",
         "dot",
         "logit",
+        "rsub",
```

Comment:
At some point we can refactor this file to follow what’s in ONNX Script so the meta data is clearer